modal_proto/api.proto

```diff
@@ -72,6 +72,14 @@ enum WebhookType {
   WEBHOOK_TYPE_WSGI_APP = 3;
 }
 
+enum WebhookAsyncMode {
+  WEBHOOK_ASYNC_MODE_UNSPECIFIED = 0;
+  WEBHOOK_ASYNC_MODE_REDIRECT = 1;  // redirect immediately to result polling endpoint
+  WEBHOOK_ASYNC_MODE_DISABLED = 2; // no redirect, fail after timeout
```

Comment:
If `DISABLED` is the default maybe it could be merged with `UNSPECIFIED = 0`, but I'm not sure about protobuf enum convention.

Reply:
It could, but I think our convention is to not use the unspecified/0-value for anything "meaningful". Good if we want to distinguish functions deployed before this change took place from those deployed after.

Reply:
Thanks, makes sense!